File: blog/src/main/kotlin/com/liux/blog/bean/po/Tag.kt
package com.liux.blog.bean.po

import com.liux.blog.bean.NoArgsConstructor
import java.io.Serializable
import java.util.*

// 标签数据
@NoArgsConstructor
data class Tag(
    // ID
    var id: Int? = null,
    // 名称
    var name: String? = null,
    // 创建时间
    var createTime: Date? = null,

    // 文章数量
    var articleCount: Int? = null,
    // 文章列表
    var articles: List<Article>? = null,
) : Serializable

File: blog/bean/api_vo/tag.go
package api_vo

import (
	"blog/bean/po"
	"time"
)

type TagVO struct {
	Id           int        `json:"id"`
	Name         string     `json:"name"`
	CreateTime   time.Time  `json:"createTime"`
	UpdateTime   *time.Time `json:"updateTime"`
	ArticleCount int        `json:"articleCount"`
}

func (v *TagVO) From(tag po.Tag) {
	v.Id = tag.Id
	v.Name = tag.Name
	v.CreateTime = tag.CreateTime
	v.UpdateTime = tag.UpdateTime
	v.ArticleCount = tag.ArticleCount
}
